lib/elixir/test/elixir/code_formatter/comments_test.exs

```diff
@@ -1262,4 +1262,98 @@ defmodule Code.Formatter.CommentsTest do
       assert_format bad, good
     end
   end
+
+  describe "defstruct" do
+    test "has first field comments" do
+      bad = ~S"""
+      defmodule Foo do
+        # defstruct
+        defstruct [ # foo
+          # 1. one
+          one: 1, # 2. one
+          # 1. two
+          # 2. two
+          two: 2
+        ]
+      end
+      """
+
+      good = ~S"""
+      defmodule Foo do
+        # defstruct
+        # foo
+        defstruct [
+          # 1. one
+          # 2. one
+          one: 1,
+          # 1. two
+          # 2. two
+          two: 2
+        ]
+      end
+      """
```

Comment:
What about:

```
        defstruct([
          # 1. one
          # 2. one
          one: 1,
          # 1. two
          # 2. two
          two: 2
        ])
```

In this case, we can sleep the container. Maybe we should pass `skip_parens?` to the function as well and always skip the brackets if it is a keyword and we are not skipping parens?

Reply:
Thank you for your review!
I fixed it , and it will be formatted as before
```elixir
        defstruct(
          # 1. one
          # 2. one
          one: 1,
          # 1. two
          # 2. two
          two: 2
        )
```